src/events/PollStartEvent.ts

```diff
@@ -188,9 +188,16 @@ export class PollStartEvent extends ExtensibleEvent<M_POLL_START_EVENT_CONTENT>
                     question: {[M_TEXT.name]: question},
                     kind: (kind instanceof NamespacedValue) ? kind.name : kind,
                     max_selections: maxSelections,
-                    answers: answers.map((a, i) => ({id: `${i + 1}-${a}`, [M_TEXT.name]: a})),
+                    answers: answers.map(a => ({id: makeId(), [M_TEXT.name]: a})),
                 },
             },
         });
     }
 }
+
+const LETTERS = "abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789";
+function makeId() {
```

Comment:
The js-sdk has a random string utility we should be using instead.

Reply:
Should we depend on js-sdk then?

Reply:
I completely forgot this is a whole other repo. Will mull it over.

Reply:
This is probably fine for now - we should consider a general utility library for element-web & its dependencies, but that's a much larger discussion than this function's existence.